doc/ansible/dev/dependent_watches.md

```diff
@@ -30,4 +30,117 @@ This is an example of a watches file with the `watchDependentResources` field se
   manageStatus: False
   watchDependentResources: True
 
-```
\ No newline at end of file
+```
+
+### Make existing resources dependent
+
+It is strongly recommended to use dependent watches, resource dependency
+should not be an issue for normal operations. However, if your operator
+deployed resources while [dependent watches were
+disabled](https://github.com/operator-framework/operator-sdk/blob/master/doc/ansible/dev/advanced_options.md#turning-off-dependent-watches-and-owner-reference-injection)
+, this section will show you how to fix it manually (using memcached as
+ the example).
+
+#### Owner Information
+
+First, we retrieve the information that we will later add to the
+dependent resources.
+
+`$ kubectl get memcacheds.cache.example.com -o yaml`
+
+```yaml
+- apiVersion: cache.example.com/v1alpha1
+  kind: Memcached
+  metadata:
+    ...(snip)
+    name: example-memcached
+    uid: ad834522-d9a5-4841-beac-991ff3798c00
+```
+
+#### Namespaced Resources
+
+For namespaced resources, we add an `ownerReference`.
```

Comment:
Can you add a short section here that just explicitly explains that the ownerReference is made up of the fields above, before showing it on a resource? ie, `The ownerReference is made up of...`